pkg/reconciler/revision/background_test.go

```diff
@@ -238,26 +240,27 @@ func TestRateLimitPerItem(t *testing.T) {
 		<-done
 	}()
 
-	start := time.Now()
 	revision := rev("rev", "img1", "img2")
-	for i := 0; i < 5; i++ {
+	for i := 0; i < 2; i++ {
```

Comment:
is there not a danger of false positives here since the amount of extra latency we're expecting from the per-item backoff is only in the 10s of milliseconds, or am I misreading? in other words I don't think we'll flake, but would the test actually consistently fail (as it should) if we weren't exponentially backing off any more?

Reply:
Just tested by exchanging the rate limiter to `DefaultControllerRateLimiter`, and that caused the test to fail immediately with

```
latency = 5.176997ms, want at least 10ms
```

The progression will currently be: 10ms, 20ms. I guess we can add a third or forth progression to add 40ms and 80ms. That should ensure exponentiality. The current one won't necessarily, that's right. A linear one would fulfill that as well.

Reply:
yeah I think this test was explicitly because we had a bug that per-item _didnt_ back off exponentially, so at least 1 more iteration would be good to confirm that stays fixed

Reply:
yep, pushed.